store/core.go

```diff
@@ -86,21 +86,70 @@ func (s *store) GetPropertyValues(ctx context.Context,
 		in.Limit = util.BtCacheLimit
 	}
 
+	direction := in.GetDirection()
+	var inArc, outArc bool
+	switch direction {
+	case "in":
+		inArc = true
+		outArc = false
+	case "out":
+		inArc = false
+		outArc = true
+	default:
+		inArc = true
+		outArc = true
+	}
```

Comment:
case "":
      inArc = true
      outArc = true
  default:
      return "Invalid value for 'direction'"